src/llm_vm/completion/optimize.py

```diff
@@ -191,7 +206,7 @@ def __init__(self, storage=local_ephemeral(), MIN_TRAIN_EXS = 20, MAX_TRAIN_EXS
         self.call_small = call_small
         self.call_big = call_big
         self.openai_key = openai_key
-        self.data_synthesizer = data_synthesis.DataSynthesis(0,100)
+        self.data_synthesizer = data_synthesis.DataSynthesis(0,50)
```

Comment:
it does seem like we more often get around that number of responses, should our magic constant here be a kw-arg?

Reply:
Maybe we should add it as a kwarg. Since max tokens depends on the model in use

Reply:
What does the variance param do? didn't see it being used in the data_synthesis method

Reply:
@abhigya-sodani do you remember what the variance parameter is? is it supposed to be like temperature or like a threshold for examples needing to be sufficiently different?

Reply:
Threshold for examples being different, I don't think we have solved a way to implement it though

Reply:
What if we implement it as the threshold for semantic similarity?

Reply:
Whatever is the fastest way to get something actually working :-)

Reply:
Great. Since temperature is already passed in as a kwarg, I'll implent the variance as a threshold parameter for the cosine similarity function